reana_commons/utils.py

```diff
@@ -52,6 +52,16 @@ def click_table_printer(headers, _filter, data):
         click.echo(formatted_output.format(*row))
 
 
+def get_user_workflows_dir(user):
```

Comment:
This method has been removed in favor of the Users model one: `user_1.get_user_workspace()`. See in [reana-server](https://github.com/reanahub/reana-server/blob/47471d5d58cea0ce78ef73068e10e0199e652349/reana_server/cli.py#L94).

Reply:
Ah ok, then I'll to change it where it's used as well in r-w-c

Reply:
@dinosk still needs to be removed